app/src/layouts/tabular/index.ts

```diff
@@ -226,6 +228,43 @@ export default defineLayout<LayoutOptions, LayoutQuery>({
 							});
 
 							description = fieldNames.join(' -> ');
+
+							const types = relationsStore.getRelationTypes(collection.value!, field.key);
+
+							if (types.at(-1) === 'o2m') {
```

Comment:
I know this is way after the fact, but it's very smelly to do this exclusively for the tabular layout. This logic shouldn't exist here